bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -382,6 +389,20 @@ public Collection<WorkItem> prRun(Path scratchPath) {
                 return List.of(new CheckWorkItem(bot, prId, errorHandler, prUpdatedAt));
             }
 
+            // Check Force push
+            if (!pr.isDraft()) {
+                var lastForcePushTime = pr.lastForcePushTime();
+                if (lastForcePushTime.isPresent()) {
+                    var lastForcePushSuggestion = pr.comments().stream()
```

Comment:
`pr.comments()` have already been fetched at the top, so no need to fetch them again.
```suggestion
                    var lastForcePushSuggestion = comments.stream()
```